TWLight/applications/models.py

```diff
@@ -319,16 +318,24 @@ def update_app_status_on_save(sender, instance, **kwargs):
             instance.date_closed = localtime(now()).date()
             instance.days_open = 0
 
-# Authorize editor to access resource after an application is saved as approved
-# or sent. We need to use post_revision_commit from django-reversion to get the
-# **current** revision user. Just listening to the post_save signal will grab
-# the user from the previously saved version.
+# Authorize editor to access resource after an application is saved as sent.
 @receiver(post_save, sender=Application)
 def post_revision_commit(sender, instance, **kwargs):
-    if reversion.is_active() and (instance.status == 2 or instance.status == 4) and not instance.imported:
```

Comment:
I remember now that I wasn't sure which state we wanted to key off of, because the authorization is a logical step that should be triggered by the approval, and be followed by communication about the approval, right?

The check for an existing authorization is definitely a thing that needs to be here, so thanks for that!

Reply:
Basically, I *think* we need to be creating auths from Approved status instead of sent.

Reply:
Per our conversation, we've really morphed our usage of `SENT` to the point where semantically it's something more akin to `FINALIZED`.